ClassNotFoundException when mapping com.google.protobuf.Timestamp

## Whats your runtime?
* Dozer version: 6.1.0
* OS version: osx 10.12.6
* JDK version: 1.8.144
  
## Whats the problem?
_Steps to reproduce:_
1. Create new dozer mapper
2. (Optional) Create one custom converter with LocalDate and com.google.protobuf.Timestamp
3. Call : mapper.map(new LocalDate(), Timestamp.class)

Dozer is trying to load class: com.google.protobuf.TimestampProto.Timestamp, which doesn't exist in proto3
 
ProtoUtils.java@156
`
return MappingUtils.loadClass(StringUtils.join(
                getFullyQualifiedClassName(descriptor.getMessageType().getFile().getOptions(), descriptor.getMessageType().getName()), '.'), beanContainer);
`

Method: 
`
private static String[] getFullyQualifiedClassName(DescriptorProtos.FileOptions options, String name) {
        return new String[] {
            options.getJavaPackage(),
            options.getJavaOuterClassname(),
            name};
    }
  `
### Observed Results:
`
Caused by: org.dozer.MappingException: java.lang.ClassNotFoundException: com.google.protobuf.TimestampProto.Timestamp from [Module "xxx:main" from local module loader @768b970c (finder: local module finder @5a4041cc (roots: /apps/jboss/modules,/apps/jboss/modules/system/layers/base))]
	at org.dozer.util.MappingUtils.throwMappingException(MappingUtils.java:82) [dozer-core-6.1.0.jar:6.1.0]
	at org.dozer.util.DefaultClassLoader.loadClass(DefaultClassLoader.java:46) [dozer-core-6.1.0.jar:6.1.0]
	at org.dozer.util.MappingUtils.loadClass(MappingUtils.java:223) [dozer-core-6.1.0.jar:6.1.0]
	at com.github.dozermapper.protobuf.util.ProtoUtils.getJavaClassIgnoreRepeated(ProtoUtils.java:156) [dozer-proto-6.1.0.jar:]
	at com.github.dozermapper.protobuf.util.ProtoUtils.getJavaClass(ProtoUtils.java:125) [dozer-proto-6.1.0.jar:]
	at com.github.dozermapper.protobuf.propertydescriptor.ProtoFieldPropertyDescriptor.getPropertyType(ProtoFieldPropertyDescriptor.java:77) [dozer-proto-6.1.0.jar:]
	at org.dozer.fieldmap.FieldMap.getSrcFieldType(FieldMap.java:131) [dozer-core-6.1.0.jar:6.1.0]
	at org.dozer.loader.MappingsParser.processMappings(MappingsParser.java:113) [dozer-core-6.1.0.jar:6.1.0]
	at org.dozer.loader.CustomMappingsLoader.load(CustomMappingsLoader.java:64) [dozer-core-6.1.0.jar:6.1.0]
	at org.dozer.DozerBeanMapper.loadCustomMappings(DozerBeanMapper.java:326) [dozer-core-6.1.0.jar:6.1.0]
	at org.dozer.DozerBeanMapper.initMappings(DozerBeanMapper.java:443) [dozer-core-6.1.0.jar:6.1.0]
	at org.dozer.DozerBeanMapper.getMappingProcessor(DozerBeanMapper.java:305) [dozer-core-6.1.0.jar:6.1.0]
	at org.dozer.DozerBeanMapper.map(DozerBeanMapper.java:214) [dozer-core-6.1.0.jar:6.1.0]
`
  
### Expected Results:
To not load TimestampProto.Timestamp but use Timestamp.
Dozer should be able to handle 2 cases, one where the class XXXProto encapsulates XXX, and other case when XXX and XXXProto are two files like Timestamp example.

--- a/proto/src/main/java/com/github/dozermapper/protobuf/util/ProtoUtils.java
+++ b/proto/src/main/java/com/github/dozermapper/protobuf/util/ProtoUtils.java
@@ -217,6 +217,10 @@ public final class ProtoUtils {
     }
 
     private static String[] getFullyQualifiedClassName(DescriptorProtos.FileOptions options, String name) {
+        if (options.getJavaMultipleFiles()) {
+            return new String[]{options.getJavaPackage(), name};
+        }
+
         return new String[] {options.getJavaPackage(), options.getJavaOuterClassname(), name};
     }
 

--- a/proto/src/test/java/com/github/dozermapper/protobuf/functional_tests/ProtoBeansMultipleFilesMappingTest.java
+++ b/proto/src/test/java/com/github/dozermapper/protobuf/functional_tests/ProtoBeansMultipleFilesMappingTest.java
@@ -1,0 +1,70 @@
+/*
+ * Copyright 2005-2017 Dozer Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.dozermapper.protobuf.functional_tests;
+
+import com.github.dozermapper.protobuf.vo.proto.NestedObject;
+import com.github.dozermapper.protobuf.vo.proto.TestObject;
+import com.github.dozermapper.protobuf.vo.protomultiple.SimpleProtoTestObject;
+import org.dozer.Mapper;
+import org.dozer.config.BeanContainer;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class ProtoBeansMultipleFilesMappingTest extends ProtoAbstractTest {
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    private Mapper mapper;
+    private BeanContainer beanContainer;
+
+    @Before
+    public void setUp() throws Exception {
+        beanContainer = new BeanContainer();
+        mapper = getMapper("mappings/protoBeansMapping.xml");
+    }
+
+    @Test
+    public void testSimpleToProto() {
+        NestedObject source = new NestedObject();
+        TestObject testObject = new TestObject();
+        testObject.setOne("ABC");
+        source.setNested(testObject);
+
+        SimpleProtoTestObject protoResult = mapper.map(source, SimpleProtoTestObject.class);
+
+        assertNotNull(protoResult);
+        assertEquals("ABC", protoResult.getNested().getOne());
+    }
+
+    @Test
+    public void testSimpleFromProto() {
+        SimpleProtoTestObject.Builder builder = SimpleProtoTestObject.newBuilder();
+        builder.getNestedBuilder().setOne("ABC");
+
+        SimpleProtoTestObject source = builder.build();
+
+        NestedObject pojoResult = mapper.map(source, NestedObject.class);
+
+        assertNotNull(pojoResult);
+        assertEquals("ABC", pojoResult.getNested().getOne());
+    }
+}

--- a/proto/src/test/java/com/github/dozermapper/protobuf/vo/proto/NestedObject.java
+++ b/proto/src/test/java/com/github/dozermapper/protobuf/vo/proto/NestedObject.java
@@ -1,0 +1,28 @@
+/*
+ * Copyright 2005-2017 Dozer Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.dozermapper.protobuf.vo.proto;
+
+public class NestedObject {
+    public TestObject getNested() {
+        return nested;
+    }
+
+    public void setNested(TestObject nested) {
+        this.nested = nested;
+    }
+
+    private TestObject nested;
+}

--- a/proto/src/test/proto/ProtoTestObjectsMultipleFiles.proto
+++ b/proto/src/test/proto/ProtoTestObjectsMultipleFiles.proto
@@ -1,0 +1,11 @@
+syntax = "proto3";
+option java_package = "com.github.dozermapper.protobuf.vo.protomultiple";
+option java_multiple_files = true;
+
+message TestObject {
+    string one = 1;
+}
+
+message SimpleProtoTestObject {
+    TestObject nested = 1;
+}

